app/scripts/controllers/preferences.test.js

```diff
@@ -869,21 +869,21 @@ describe('preferences controller', function () {
       );
     });
   });
-  describe('setUseStaticTokenList', function () {
+  describe('setUseTokenDetection', function () {
     it('should default to false', function () {
       const state = preferencesController.store.getState();
-      assert.equal(state.useStaticTokenList, false);
+      assert.equal(state.useTokenDetection, true);
     });
 
-    it('should set the useStaticTokenList property in state', function () {
+    it('should set the useTokenDetection property in state', function () {
       assert.equal(
-        preferencesController.store.getState().useStaticTokenList,
-        false,
+        preferencesController.store.getState().useTokenDetection,
+        true,
       );
-      preferencesController.setUseStaticTokenList(true);
+      preferencesController.setUseTokenDetection(false);
       assert.equal(
-        preferencesController.store.getState().useStaticTokenList,
-        true,
+        preferencesController.store.getState().useTokenDetection,
+        false,
```

Comment:
same with all the values in this test.

Reply:
The default value is `true`, so the test is checking if it is getting set to `false` now but using the function,